src/mmap_unix.rs

```diff
@@ -278,6 +342,19 @@ mod tests {
         }
     }
 
+    #[test]
```

Comment:
Sure, if decreasing the coverage it's fine, we can drop it.

Reply:
Coverage should be there just so that we get a better idea when we forgot to add tests. I am pretty against coverage tests, but I can understand how it can be rather a subjective view. I would drop it if you're also fine with that.